### DISCUSSION TITLE
Is Version 2.3.1 released?

### DISCUSSION THREAD

**Discussion by noelmccrory** [awspring/spring-cloud-aws#115] [General]
Hi,

I see on Maven Central that v2.3.1 is available since 1st April. However, there is no release tag or notes on GitHub. Can you confirm that v2.3.1 is officially released and if possible supply release notes?

Thanks

**Comment by maciejwalkowiak** [comment]
Hi! Sorry for a delay. Yes, 2.3.1 is released: https://github.com/awspring/spring-cloud-aws/releases/tag/v2.3.1
